testing/font-comic-neue/APKBUILD

```diff
@@ -0,0 +1,49 @@
+# Contributor: Orson Teodoro <orsonteodoro@hotmail.com>
+# Maintainer: Orson Teodoro <orsonteodoro@hotmail.com>
+pkgname=font-comic-neue
+_pkgname=comicneue
+pkgver=20160917
+pkgrel=0
+pkgdesc="Comic Neue is a font that fixes the shortcomings of Comic Sans"
+url="http://comicneue.com/"
+arch="noarch"
+license="OFL-1.1"
+depends="fontconfig"
+makedepends=""
+commit="216e1cca320e7e7886c9523f9afbb7972eb7a9f2"
+source="$_pkgname-$pkgver.zip::https://github.com/crozynski/comicneue/archive/$commit.zip"
+builddir="$srcdir/$_pkgname-$commit"
+subpackages="$pkgname-doc"
+options="!check"
+
+build() {
+	cd "$builddir"
+	return 0
+}
+
+package() {
+	cd "$builddir"
+	mkdir -p "$pkgdir/usr/share/fonts/$_pkgname"
+
+	for f in $(find OTF -type f -name "*.otf"); do
+		mv "$f" "$pkgdir"/usr/share/fonts/$_pkgname
+	done
+}
+
+doc() {
+	cd "$builddir"
+
+	msg "Packaging documentation"
+
+	mkdir -p "$subpkgdir"/usr/share/doc/$_pkgname/
+	cp -a FONTLOG.txt		"$subpkgdir"/usr/share/doc/$_pkgname/ || return 1
+	cp -a Booklet-ComicNeue.pdf	"$subpkgdir"/usr/share/doc/$_pkgname/ || return 1
+	cp -a README.md			"$subpkgdir"/usr/share/doc/$_pkgname/ || return 1
```

Comment:
This is unnecessary verbose, just do this:

```sh
	cp FONTLOG.txt Booklet-ComicNeue.pdf README.md \
		"$subpkgdir"/usr/share/doc/$_pkgname/
```

Also explicit `|| return 1` is not needed anymore (abuilds run with `set -e`), please remove them.